pagopa/io-eudiw-app
# feat: [EUDIW-163,EUDIW-167] Add navigation and store structure

> [!NOTE] 
> Depends on #43 

## Short description
This PR adds a basic navigation and store structure, using the onboarding process as example.

## List of changes proposed in this pull request
- Add a main navigation, a tab navigation and an onboarding navigation;
- Setup both navigation and store to have types;
- Update locales;
- Update `eslint` rules to avoid importing store hook without types;
- Restore the previous `README.md`.

## How to test
- Open the app and press on "Finish onboarding". This should bring you the main screen with the tab navigator; 
- Close the app and reopen it. You should still see the main screen with the tab navigator; 
- Press on "Reset onboarding" and you should see again the initial screen with the "Finish onboarding" label.
